src/services/highlight.tsx

```diff
@@ -104,19 +110,22 @@ export const highlightValueStringMatches = (
 };
 
 // @todo cache results by regex/value?
-export const getLineFilterMatches = (matchExpressions: RegExp[], value: string): Array<[number, number]> => {
+export const getLineFilterMatches = (
+  matchExpressions: Array<RegExp | undefined>,
+  value: string
+): Array<[number, number]> => {
   let results: Array<[number, number]> = [];
   matchExpressions.forEach((regex) => {
     let valueMatch;
     let valueMatches = [];
     do {
       try {
-        valueMatch = regex.exec(value);
+        valueMatch = regex?.exec(value);
         if (valueMatch) {
           valueMatches.push(valueMatch);
         }
       } catch (e) {
-        logger.error(e, { msg: 'Error executing match expression', regex: regex.source });
+        logger.error(e, { msg: 'Error executing match expression', regex: regex?.source ?? '' });
```

Comment:
these might be too noisy, maybe a warning or an info would be better?

Reply:
Maybe only enable by a debug flag.

Reply:
Plugin feature flags would be lovely so we can have different logging levels for each env. I downgraded to an info for now.